### DISCUSSION TITLE
Kidney Intracellular Protein Concentration Calculation

### DISCUSSION THREAD

**Discussion by jhunt8** [Open-Systems-Pharmacology/Forum#1158] [Questions & Problems] [answered]
In PKSim v9, the simulation parameter "Kidney|Intracellular|volume of protein container" retains the value of the whole kidney volume, while the parameter "Kidney|Intracellular|Volume" is calculated from the product of "Fraction intracellular" and total kidney volume. Each is used to calculate intracellular transporter (efflux, apical) and intracellular enzyme concentrations differently than calculations for interstitial (influx) transporter concentrations. Why is this? Thank you!

The attached .txt file shows annotated examples from PKSim "structure to file" export. 

[Kidney_Intracellular_Protein_Concentration_Calculations_StructuretoFile.txt](https://github.com/Open-Systems-Pharmacology/Forum/files/8188760/Kidney_Intracellular_Protein_Concentration_Calculations_StructuretoFile.txt)

**Comment by StephanSchaller** [comment]
I believe expression data is normally derived from whole tissue samples  which is why the total organ volume is used but the amount is then allocated to the site of transporter expression.
@PavelBal can likely link the specifications here which should explain how this was implemented.

**Reply by jhunt8** [reply]
Thank you!

**Comment by PavelBal** [comment]
There is no specification available how it was implemented in V9.

Basically, the statement "relative expression of 100% with reference concentration 1 µmol/l" means that the whole organ (inkl. plasma and RBC) has the particular concentration. The amount in the whole organ is therefore "1 µmol/l times volume of whole organ", and the concentration IN intracellular space is "1 µmol/l times volume of whole organ divided by intracellular volume".

For the interstitial space, this has not been taken into account and was a bug in the implementation. Version 10 corrects this shortcoming, the specification of how protein expression is calculated in V10 can be found https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data

**Reply by jhunt8** [reply]
Thank you!

**Reply by jhunt8** [reply]
Thank you both again for the answers! The increased specificity of protein localization in v10 is excellent. @PavelBal, can I update the simulation structure, or do I need to rebuild the simulation in v10? Apologies if interfacing [releases](https://github.com/Open-Systems-Pharmacology/Suite/releases) is documented elsewhere.

**Comment by jhunt8** [comment]
Thank you again for your great answers @StephanSchaller and @PavelBal! I'll post what I've found so far. The bug can be corrected by hand in MoBi. If I'm understanding correctly, the easiest way to do this is to add another "...|" to the parameter path so that "Volume" calls the volume of the organ instead of the interstitial compartment. 
<img width="752" alt="Interstitial_Path" src="https://user-images.githubusercontent.com/87446546/161597478-895dd45d-54be-439f-a09c-be040a299fda.PNG">
I haven't confirmed that this bug is corrected automatically in a new v10 project, but unless I'm misunderstanding, it isn't automatically corrected when a project created in v9 is opened in v10. Thank you again for your explanations!

**Comment by PavelBal** [comment, marked as answer]
> If I'm understanding correctly, the easiest way to do this is to add another "...|" to the parameter path so that "Volume" calls the volume of the organ instead of the interstitial compartment.

That's correct.

> I haven't confirmed that this bug is corrected automatically in a new v10 project, but unless I'm misunderstanding, it isn't automatically corrected when a project created in v9 is opened in v10. Thank you again for your explanations!

This is only corrected when you re-create your project in PK-Sim v10 (e.g. using the snapshot function (https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/importing-exporting-project-data-models#exporting-project-to-snapshot-loading-project-from-snapshot). MoBi does not do any conversions in the models (or only in very rare cases).
